docs/experiment/general/experiment-duration-estimate.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Experiment Duration Estimate
+description: Understand how experiment duration estimate is calculated
+---
```

Comment:
example help doc of competitor https://help.vwo.com/hc/en-us/articles/900004766743-Estimate-the-Duration-to-Get-Statistically-Significant-Results-From-Your-Test-in-VWO